our_files/custom.js

```diff
@@ -1115,66 +1222,102 @@ var aeonRequest = function() {
     encodeURIComponent(
       jQuery("#detail0_ACCESSRESTRICTIONS .ACCESSRESTRICTIONS_value").text()
     );
-  setTimeout(function() {
-    jQuery(
-      ".detailItemsDiv .detailItemTable > tbody > tr.detailItemsTableRow"
-    ).each(function() {
-      var libr = jQuery(this)
-        .find(".asyncFieldLIBRARY")
-        .first()
-        .text();
-      var itemDocType =
-        "&DocumentType=" +
-        encodeURIComponent(
-          jQuery(this)
-            .find(".detailItemsTable_ITYPE")
-            .text()
-            .replace(/\n/g, "")
-        );
-      var itemCall =
-        "&CallNumber=" +
-        encodeURIComponent(
-          jQuery(this)
-            .find(".detailItemsTable_CALLNUMBER")
-            .text()
-            .replace(/\n/g, "")
-        );
-      var curLocation = jQuery(this)
-        .find(".asyncFieldSD_ITEM_STATUS")
-        .first()
-        .text();
-      var itemLocation = "&Location=" + encodeURIComponent(curLocation);
-      if (libr == SPEC_COLL || libr == ALT_SPEC_COLL) {
-        if (curLocation == REMOTE) {
-          requestType = "&Value=GenericRequestAllIronMountain";
-        } else {
-          requestType = "&Value=GenericRequestAll";
+  aeonIntervalPlaceholder = setInterval(function() {
+    jQuery(".detailItemsDiv .detailItemTable > tbody > tr.detailItemsTableRow")
+      .add(".detailItems .detailItemTable .detailChildRecord")
+      .each(function() {
+        const mobileMOdifier = jQuery(this).hasClass("detailChildRecord")
```

Comment:
Doing this simplified the calls to those elements - mobile had an additional class for labels and values